bin/ci/http_server

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -ex
+
+java -cp target/http-server-1.0-SNAPSHOT.jar server.Main 5000 /Users/Travis/Development/8th-Light/Apprenticeship/projects/resources/http-server/public
```

Comment:
couldn't this be relative so it doesn't have your dir info in it

Reply:
I spent a little time trying to figure out how to get relative paths to work and couldn't crack it. I would prefer it though--I'll look into it some more.

Reply:
Yeah, cause right now it won't work for anyone but you...

what have you tried thus far? Could you change code to take relative and use file tools to find absolute?

Reply:
Which would be why my CI builds are failing! I'm looking into that right now, thanks!

Reply:
That did it! Thanks!